### ISSUE TITLE
Performance bugs

### ISSUE THREAD

**Issue by armanogan** [claviska/jquery-minicolors#276]
Hey guys,

I wondered,why you are attaching a lot of  events mousedown,when you can use only 1and your mousemove and mouseup  events are working always even if there is no element attached. You need to attache mousemove and mouse ONLY after mousedown event and remove the events on mouse up.The plugin works good but the code dirty  is dirty.

**Comment by claviska** [comment]
You're right about the `mousemove` handler firing too frequently — it should only fire when the color picker is open. However, your observation isn't presented very tactfully.

Since you obviously took more than a passing glance at the code, why didn't you bother to submit a PR?

**Comment by armanogan** [comment]
I have already re-coded as it should be, if you want you can take a look.


[jquery.minicolors.zip](https://github.com/claviska/jquery-minicolors/files/3227924/jquery.minicolors.zip)

**Comment by kolosovsky** [comment]
I have the same issue. Mousemove event triggers angular change detection and therefore causes performance issues.
